src/cheatsheets/gleam-for-erlang-users.md

```diff
@@ -192,3 +192,163 @@ fn main() {
 }
 ```
 
+## Data types
+
+### Strings
+
+All strings in Gleam are UTF-8 encoded binaries.
+
+#### Erlang
+
+```erlang
+<<"Hellø, world!"/utf8>>.
+```
+
+#### Gleam
+
+```rust,noplaypen
+"Hellø, world!"
+```
+
+### Tuples
+
+Unlike in Erlang, there is no tuple literal syntax in Gleam, but they are very useful in Gleam as
+they're the only collection data type that allows for mixed types of elements in the collection.
```

Comment:
This isn't quite right, we have a tuple literal syntax in Gleam! You've used it below :)

Reply:
That looks a whole lot like a function call and not a literal... I guess there‘s no a reason why that can’t be a literal, but to someone just coming in to the language that would surely be mistaken as a function call!

Reply:
It's a similar syntax, but it's not a function. You can't assign the `tuple` keyword to a variable, for example.